v1.1/show-jobs.md

```diff
@@ -4,35 +4,57 @@ summary: The SHOW JOBS statement lists all currently active schema changes and b
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> The `SHOW JOBS` [statement](sql-statements.html) lists details about currently active schema changes and backup/restore jobs, including:
+<span class="version-tag">New in v1.1:</span> The `SHOW JOBS` [statement](sql-statements.html) lists all of the types of long-running tasks your cluster has performed, including:
 
-- xx
-- xx
-- xx
+- Schema changes through `ALTER TABLE`
+- Enterprise [`BACKUP`](backup.html), [`RESTORE`](restore.html), and `IMPORT`
```

Comment:
Should this also link to the import table docs?

Reply:
Yes, @sploiselle, since there's already a page for `IMPORT TABLE`, let's link there.